ide/nativeimage.api/src/org/netbeans/modules/nativeimage/api/debug/EvaluateException.java

```diff
@@ -16,17 +16,24 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.netbeans.modules.cpplite.debugger;
```

Comment:
Copied from cpplite? Should the `cpplite` debugger exception be replaced by this one?

Reply:
It is moved.